src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from 'react-router-dom';
+
+export function Button({
+  className = 'btn',
+  text = '',
+  href = '',
```

Comment:
Maybe no default parameter for `href`, then down below you can just check if `href` exists.

Reply:
I removed default values for `text` & `href`. They were similar. 😄